cmd/xpumanager_sidecar/README.md

```diff
@@ -0,0 +1,68 @@
+# XeLink sidecar for Intel XPU Manager
+
+Table of Contents
+
+* [Introduction](#introduction)
+* [Modes and Configuration Options](#modes-and-configuration-options)
+* [Installation](#installation)
+  * [Install XPU-Manager with the Sidecar](#install-xpu-manager-with-the-sidecar)
+  * [Install Sidecar to an Existing XPU-Manager](#install-sidecar-to-an-existing-xpu-manager)
+* [Verify Sidecar Functionality](#verify-sidecar-functionality)
+
+## Introduction
+
+Intel GPUs can be interconnected via an XeLink. In some workloads it is beneficial to use GPUs that are XeLinked together for optimal performance. XeLink information is provided by [Intel XPU Manager](https://www.github.com/intel/xpumanager) via its metrics API. Xelink sidecar retrieves the information from XPU Manager and stores it on the node under ```/etc/kubernetes/node-feature-discovery/features.d/``` as a feature label file. NFD reads this file and stores the contents as node labels. These labels are then used by [GAS](https://github.com/intel/platform-aware-scheduling/tree/master/gpu-aware-scheduling) to make [scheduling decisions](https://github.com/intel/platform-aware-scheduling/blob/master/gpu-aware-scheduling/docs/usage.md#multi-gpu-allocation-with-xe-link-connections) for Pods.
+
+## Modes and Configuration Options
+
+| Flag | Argument | Default | Meaning |
+|:---- |:-------- |:------- |:------- |
+| -lane-count | int | 4 | Minimum lane count for an XeLink interconnect |
+| -interval | int | 10 | Interval for XeLink topology fetching and label writing (seconds, >= 1) |
+| -startup-delay | int | 10 | Startup delay before the first topology fetching (seconds, >= 0) |
+| -label-namespace | string | gpu.intel.com | Namespace for the labels |
+
+The sidecar also accepts a number of other arguments. Please use the -h option to see the complete list of options.
+
+## Installation
+
+The following sections detail how to obtain, deploy and test the XPU-Manager XeLink sidecar.
+
+### Pre-built Images
+
+[Pre-built images](https://hub.docker.com/r/intel/intel-xpumanager-sidecar)
+of this component are available on the Docker hub. These images are automatically built and uploaded
+to the hub from the latest main branch of this repository.
+
+Release tagged images of the components are also available on the Docker hub, tagged with their
+release version numbers in the format `x.y.z`, corresponding to the branches and releases in this
+repository. Thus the easiest way to deploy the plugin in your cluster is to run this command
+
+Note: Replace `<RELEASE_VERSION>` with the desired [release tag](https://github.com/intel/intel-device-plugins-for-kubernetes/tags) or `main` to get `devel` images.
+
+See [the development guide](../../DEVEL.md) for details if you want to deploy a customized version of the plugin.
+
+#### Install XPU-Manager with the Sidecar
+
+Install XPU-Manager daemonset to nodes that have **i915_monitoring** resource with the XeLink sidecar
+
```

Comment:
Kustomize file seems to drop other resources than  `i915_monitoring`, which is not not clear from this description.  IMHO it would be clearer just as:
> Install XPU-Manager daemonset with the XeLink sidecar



There are also several other things that are needed for XPUM to be useful, starting from Prometheus.   There could be e.g. a link to XPUM project instructions, that tells what is needed for fully functional installation.

Reply:
For the sidecar Prometheus doesn't bring anything. But I'll add a link to the XPUM kubernetes documentation.